R/score.R

```diff
@@ -0,0 +1,57 @@
+# score.R
+
+#' Produce a gH object: a hash of genes and their associated scores
+#' (likelihood that observed mutations exceed background mutations at FDR of 1%)
+#'
+
+score <- function(gX, gH, silent = FALSE, noLog = FALSE) {
+    
+    check parameters with .checkArgs()
+    
+    open gX file
+    
+    # for each gene, check if it already in the gH hash and if yes, do not calculate its score (in case have a repeat of a gene in gX, would just use first calculated score)

```

Comment:
I don't agree with dropping data. And anyway: have you looked at how a gX is structured? I think you are misunderstanding this: every gene may have **many** records of changes included. They are sorted for convenience, but they don't really have to be.